test/scripts/test-wrapper.go

```diff
@@ -18,7 +18,7 @@ import (
 	testutils "github.com/mesosphere/kubeaddons/test/utils"
 )
 
-const (
+var (
 	upstreamRemote = "origin"
 	upstreamBranch = "master"
 )
```

Comment:
nit: can you make these consts again and call them `defaultUpstreamRemote` etc (mutable globals are bleugh - yeah i know this is just a small utility but still...)

```suggestion
const (
	defaultupstreamRemote = "origin"
	defaultUpstreamBranch = "master"
)
```